src/rtsp_player_controller.cc

```diff
@@ -125,17 +126,19 @@ void RTSPPlayerController::InitPlayer(const std::string& url,const double& audio
 	need_audio_data_ = false;
 	audio_level_cb_frequency_ = audio_level_cb_frequency;
 
-	player_thread_->message_loop().PostWork(cc_factory_.NewCallback(&RTSPPlayerController::InitializeStreams, url));
+	player_thread_->message_loop().PostWork(cc_factory_.NewCallback(
+		&RTSPPlayerController::InitializeStreams, url, arlo_crt_path));
 }
 
 void RTSPPlayerController::UpdateAudioConfig() {
+	AVStream* s = format_context_->streams[audio_stream_idx_];
 	//Hardcode settings to AAC
 	audio_config_.codec_type = Samsung::NaClPlayer::AUDIOCODEC_TYPE_AAC;
 	audio_config_.codec_profile = Samsung::NaClPlayer::AUDIOCODEC_PROFILE_AAC_LOW;
 	audio_config_.channel_layout = Samsung::NaClPlayer::CHANNEL_LAYOUT_MONO;
 	audio_config_.sample_format = Samsung::NaClPlayer::SAMPLEFORMAT_PLANARF32;
-	audio_config_.bits_per_channel = 6; // bitrate divided by sample rate
-	audio_config_.samples_per_second = 8000;
+	audio_config_.bits_per_channel = s->codecpar->bit_rate / s->codecpar->sample_rate;
+	audio_config_.samples_per_second = s->codecpar->sample_rate;
```

Comment:
we don't really want to do this (I did it for testing, it avoids this assert failing: https://github.com/PhysicalGraph/STAVPlay/blob/master/src/transcode_utils.h#L88), it just sets bits_per_channel and samples_per_second to 0. we need to properly handle different types of streams. Arlo audio is 1600hz AAC, which is very different from the Techwin PCM format. I believe we still need to decode the Arlo AAC to do audio level metering? @maheshrpb